spotipy/client.py

```diff
@@ -224,6 +229,9 @@ def _internal_call(self, method, url, payload, params):
             if payload:
                 args["data"] = json.dumps(payload)
 
+        if self.language is not None:
+            headers["Accept-Language"] = '%s;q=1' % self.language
```

Comment:
I'm trying to find what `%s;q=1` is.  Any chance you can add a very short comment to give a reference?

Reply:
The q value indicates the relative priority if multiple language are specified, but since only one language is used here, I think I'd better remove it for readability. If you think this is okay, I will send a revised version and include the changelog as well.

```suggestion
            headers["Accept-Language"] = self.language
```

Reply:
alright, yes it looks good without